src/core/constraint_template.jl

```diff
@@ -32,5 +50,4 @@ function constraint_voltage_magnitude_sqr_on_off(pm::_PMs.GenericPowerModel, i::
     bus = _PMs.ref(pm, nw, :bus, i)
 
     constraint_voltage_magnitude_sqr_on_off(pm, nw, cnd, i, bus["vmin"][cnd], bus["vmax"][cnd])
-end
-
+end
```

Comment:
It is considered a best practice that every line of code should end with a line terminator `\n`, which would result in having a blank line at the end of the file (see https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

Reply:
Fixed.  Was this the only file you this on?

Reply:
Looking at the Files Changed, I think `runtests.jl` also has this issue

Reply:
Fixed, along with a couple other files (PowerModelsMLD.jl, ac-mld-uc.jl, runtests.jl)